firestore/cloud-client/snippets.py

```diff
@@ -209,10 +209,14 @@ def add_data_with_id():
 
 
 def add_custom_class_generated_id():
-    db = firestore.Client()
+    db = firestore.Client(project="maratt")
```

Comment:
I think you need to revert this to get the sample to pass

Reply:
Ah yes, mistakenly committed that. I've reverted this. thanks